packages/editor/editor.css

```diff
@@ -32,6 +32,7 @@ https://alex.dytry.ch/toys/palette/?palette=%7B%22families%22:%5B%22black%22,%22
 	/* These cursor values get programmatically overridden */
 	/* They're just here to help your editor autocomplete */
 	--tl-cursor: var(--tl-cursor-default);
+	--tl-cursor-chat-placeholder: 'Type to chat...';
```

Comment:
If this is going to be a CSS var will we still be able to localise it?

Reply:
This gets overriden by JS with localised text. I put the english text there as a placeholder to indicate what sort of thing it gets set to. But this might not be totally clear ¯\\_(ツ)_/¯

We could maybe change it to an empty string instead, which might be clearer!

Reply:
oh this makes sense! i think part of this is me misunderstanding the comment and thinking it only applied to `--tl-cursor`